packages/carbon-graphs/tests/unit/controls/Graph/GraphResize-spec.js

```diff
@@ -586,4 +586,27 @@ describe('Graph - Resize', () => {
       expect(graph.config.canvasHeight).toBe(250);
     });
   });
+  describe("Only when graph container is present", () => {
+    it("Resize the graph", (done) => {
```

Comment:
nit: `should resize the graph`

Reply:
Updated [here](https://github.com/JananiGunasekaran/terra-graphs/commit/b4b9e470b74001a11534a3d7d492a2c1f10c8d52).